### PR TITLE
subscription block

### LINKED ISSUE #25: subscription block

### PR DESCRIPTION
**Pull request by sneharora1**
Fix #25

Test URLs:
- Before: https://main--infosys--aemsites.hlx.page/drafts/sneh/subsription
- After:https://subscription--infosys--aemsites.hlx.page/drafts/sneh/subsription

Pending:
1. Animation while clicking on the popup divs
2. Form submission to Infosys URL

### CODE REVIEW

File: blocks/form/form-fields.js [hunk 1]
```diff
@@ -120,6 +148,13 @@ const createSubmit = (fd) => {
   button.classList.add('button');
   button.type = 'submit';
 
+  // button.addEventListener('click', (event) => {
```
Comment (shrotia) [inline blocks/form/form-fields.js:151]: please remove the commented code if not needed.

**Comment by shrotia** [comment]
@nawaz-adobe changes looks fine, some cleanup and redundant variables can be removed.
Also lets discuss if we can keep Forms submit flow intact.

File: blocks/form/form-fields.js (hunk 1 above)
Comment (nawaz-adobe) [reply blocks/form/form-fields.js]: removed

**Comment by nawaz-adobe** [comment]
> @nawaz-adobe changes looks fine, some cleanup and redundant variables can be removed. Also lets discuss if we can keep Forms submit flow intact.

@shrotia cleaned up `styles.css` 

Forms submit flow is also fixed.

File: styles/styles.css [hunk 2]
```diff
@@ -194,6 +207,16 @@ a:hover {
   color: var(--link-hover-color);
 }
 
+[class^="icon-"]::before, [class*=" icon-"], [class^="icon-"] {
```
[the remark this answers is not in the document]
Comment (shrotia) [reply styles/styles.css:213]: we already have this in icons.css, you can remove this from this PR.

**Review by shrotia — APPROVED** [review]

**Comment by shrotia** [comment]
Looks good overall. Please fix the error reported by "linting" and address the one comment. Otherwise, it's ready to merge.

File: styles/styles.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (nawaz-adobe) [reply styles/styles.css]: removed

**Pull request merged**